### ISSUE TITLE
Enabled: true on test basis

### ISSUE THREAD

**Issue by zmallen** [fastly/ftw#72]
We should have `enabled:true` on a test by test basis. Ive noticed its hard to parse WAF logs when working in a rulefile with multiple rules, and I know that other tests are passing while working on a new one.

**Comment by csanders-git** [comment]
Either that or a single rule running mode.

**Comment by zmallen** [comment]
Lets knock this out when you get to Vegas for a release

**Comment by zmallen** [comment]
#75
